css/index.css

```diff
@@ -1 +1,513 @@
+    padding: 35px 0px;
+    border-top: 2px solid black;
+    border-bottom: 2px solid black;
+    align-items: center;
+}
+
+.Sec2s1 {
+    display:flex;
+    width: 90.91%;
+    max-width: 800px;
+    justify-content: space-between;
+    align-items: center;
+}
+
+
+.s2 {
+    display:flex;
+    flex-direction:column;
+    max-width: 225px;
+    justify-content: space-between;
+}
+
+footer {
+    width: 90.91%;
+    max-width: 800px;
+    height: 200px;
+    display:flex;
+    flex-direction:column;
+    justify-content: space-between;
+    margin-top: 35px;
+}
+
+a {
+    text-decoration: none;
+    color: gray;
+}
+
+nav a {
+    font-size: 2.1rem;
+}
+
+header img {
+    width:150px;
+}
+
+h1 {
+    font-size: 4rem;
+    text-align: center;
+}
+
+.Sec1 a {
+    text-align: center;
+    font-size: 2.7rem;
+    padding: 10px 3%;
+    border: 2px solid gray;
+}
+
+.s1 p {
+    font-size: 1.9rem;
+    max-width: 355px;
+    width: 100%;
+}
+
+.Sec2 img {
+    width: 100%;
+    max-width: 880px;
+    margin: 80px 0px 40px 0px;
+}
+
+.Sec2s2 {
+    height: 225px;
+    width: 90.91%;
+    max-width: 800px;
+    display:flex;
+    justify-content: space-between;
+}
+
+.s2 p {
+    font-size: 1.9rem;
+    width: 100%;
+    max-width: 225px;
+}
+
+address {
+    font-size: 1.9rem;
+    line-height: 1.25;
+}
+
+.Copyright {
+    font-size: 1.6rem;
+    align-self: center;
+    margin: 50px 0px 10px;
+}
+
+h2 {
+    margin-bottom:30px;
+    font-size: 3rem;
+    height: 10px;
+}
+
+p {
+    line-height: 1.25;
+}
+
+@media (max-width:800px) {
+
+    .Main {
+        max-width:100%;
+        width: 800px;
+    }
+
+    .Main2 {
+        display:flex;
+        flex-direction:column;
+        align-items: center;
+        width: 100%;
+        max-width: 800px;
+    }
+
+    .Sec1 {
+        width: 90.00%;
+        max-width: 720px;
+    }
+
+    .Sec1 img {
+        display: none;
+    }
+
+    header {
+        width: 90%;
+        max-width:720px;
+        flex-direction: column;
+        height: 100px;
+        justify-content: space-around;
+        align-items: center;
+    }
+
+    nav {
+        width:100%;
+        max-width: 720px;
+        order:1;
+    }
+    
+    header img {
+        order:0;
+        width: 200px;
+    }
+
+    .Sec2 {
+        padding:0px 0px;
+        width: 100%;
+        max-width: 800px;
+
+    }
+
+    .Sec2 img {
+        width: 100%;
+        max-width: 800px;
+        margin: 30px 0px;
+    }
+
+    .Sec2s1 {
+        width: 90.00%;
+        max-width: 720px;
+        display:flex;
+        height:250px;
+        align-items: flex-start;
+        justify-content: space-between;
+        margin-top: 30px;
+    }
+
+    
+    .Sec2s2 {
+        width: 90.00%;
+        height: 480px;
+        max-width: 720px;
+        display:flex;
+        justify-content: space-between;
+        align-items: flex-start;
+    }
+
+    .s1 p {
+    font-size: 1.9rem;
+    width: 100.00%;
+    max-width: 325px;
+}
```

Comment:
The spacing is off in your CSS. If you haven't yet, consider adding the extensions `Beautify` or `Prettier` to clean up your CSS.